src/lib/buyables/tokkulBuyables.ts

```diff
@@ -0,0 +1,253 @@
+import itemID from '../util/itemID';
+
+interface TokkulShop {
+	name: string;
+	inputItem: number;
+	tokkulReturn: number;
+	buyable?: boolean;
```

Comment:
instead of explicitly making a boolean for if its buyable, just look if youve defined a sell price for it